src/Sentry.Unity/Integrations/UnityLogHandlerIntegration.cs

```diff
@@ -149,6 +152,32 @@ private void OnQuitting()
             _hub?.FlushAsync(_sentryOptions?.ShutdownTimeout ?? TimeSpan.FromSeconds(1)).GetAwaiter().GetResult();
         }
 
+
+        internal static bool PassesMinimumBreadcrumbLevel(LogType minimumLogType, LogType logType)
```

Comment:
Maybe also document the chosen older in the comment to make it easier to read? What is the order you've chosen anyway? From my naive look just from the names I'd expect the following order:

```
    Log,
    Warning,
    Exception,
    Assert,
    Error,
```

or maybe `Exception` & `Assert` swapped, but they're actually orthogonal.

Also, `Error` isn't really defined in Unity docs so it's not clear to me whether it's any worse than `Exception` or not.

Reply:
On a second thought... maybe we should give user a set of checkboxes to pick which they want to handle? So we don't have to come up with arbitrary ordering that's always going to make someone unhappy.

Reply:
Actually, this is a very fair point.